utils/types/deployment/deployment.go

```diff
@@ -20,11 +18,7 @@ const defaultClusterType = DedicatedType
 func GetFromEnv() (Type, error) {
 	t := Type(config.GetEnv("DEPLOYMENT_TYPE", ""))
```

Comment:
So we would need to mention type as multitenant in the next release  for hosted. Shall we make it backwards compatible?